js/viewmodel.js

```diff
@@ -7,7 +7,30 @@ window.FBE = window.FBE || {};
     var events = new TinyEmitter(),
         bp = new Blueprint(),
         selectedItem,
-        entityNamesThatDoNotRotate = /beacon|roboport|lab|heat_pipe|pipe$|furnace|chest|pole|substation|solar|accumulator|centrifuge|wall|rocket_silo|radar|turret|speaker|power_switch|reactor|lamp|land_mine/i
+        entityNamesThatDoNotRotate = /beacon|roboport|lab|heat_pipe|pipe$|furnace|chest|pole|substation|solar|accumulator|centrifuge|wall|rocket_silo|radar|turret|speaker|power_switch|reactor|lamp|land_mine/i,
+        defaultDirectionsByEntityName = [

```

Comment:
the variable name sounds like it will contain a map, with a EntityName being the key. `entityDefaultDirections` maybe?

Reply:
will do